subsys/net/ip/net_context.c

```diff
@@ -2335,6 +2335,11 @@ static void set_pkt_txtime(struct net_pkt *pkt, const struct msghdr *msghdr)
 static void set_pkt_hoplimit(struct net_pkt *pkt, const struct msghdr *msg_hdr)
 {
 	struct cmsghdr *cmsg;
+	const struct sockaddr_in6 *addr6 = NULL;
+
+	if (IS_ENABLED(CONFIG_NET_IPV4_MAPPING_TO_IPV6) && IS_ENABLED(CONFIG_NET_IPV6)) {
```

Comment:
Don't we need to check that the `pkt` addr family is `AF_INET6` here? This function could be called for IPv4 too.

Reply:
Ah, my bad, was reading the diff badly, sorry for the noise.